src/extensions/extension-manager.ts

```diff
@@ -131,6 +131,9 @@ export class ExtensionManager {
         continue
       }
       const absPath = path.resolve(folderPath, fileName);
+      if (!fs.existsSync(absPath) || !fs.lstatSync(absPath).isDirectory()) {
+        continue;
+      }
       const manifestPath = path.resolve(absPath, "package.json");
       await fs.access(manifestPath, fs.constants.F_OK)
```

Comment:
I think this is where @stevejr encountered an issue. We're assuming any directory here will contain an extension. For Steve there's a directory (.git) that doesn't have package.json in it. Should we catch here, log a warning, and continue?

Reply:
Also, there should probably be a similar change for lines 111-113

Reply:
Thanks for pointing to this problem. Resolved by moving checking for `package.json` into `getExtensionByManifest()`